api_tests/nodes/views/test_node_draft_registration_list.py

```diff
@@ -102,12 +102,32 @@ def draft_registration(self, user, project_public, schema):
             branched_from=project_public
         )
 
+    @pytest.fixture()
+    def project_private(self, user, user_write_contrib, user_read_contrib, group, group_mem):
```

Comment:
The draft being public causes it to always 200, so I had to add this to fit the AdminContributorOrPublic permission.

Reply:
That's not a problem with the test, it's a problem with the code. We forgot that the permissions class is checking the Node, not the Draft Registration, so all users will be able to see DraftRegistrations for a Public Project. That's certainly not what we want.

It also means your original permission class wasn't quite right, either, since (for now) DraftRegistrations and the Nodes they're based on can have separate permissions...ugh.